doc/classes/SphereMesh.xml

```diff
@@ -20,7 +20,8 @@
 			Number of radial segments on the sphere.
 		</member>
 		<member name="radius" type="float" setter="set_radius" getter="get_radius" default="0.5">
-			Radius of sphere.
+			Radius of the sphere's equator.
+			[b]Note:[/b] To get a regular sphere, the radius of the sphere must be equal to half its height.
```

Comment:
Use `'s` to show possession, as per the [guidelines](https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html):

```suggestion
			[b]Note:[/b] To get a regular sphere, the sphere's radius must be equal to half its height.
```

Reply:
I've noticed that I repeat "sphere" a bit too much for my liking in this line.

How about:
`To get a regular sphere, its radius must be equal to half its height.` ?